src/install/kafka/default-configuration.mdx

```diff
@@ -0,0 +1,48 @@
+---
+componentType: default
+headingText: Configure the Kafka integration
+---
+
+1. Change directory to the integrations configuration folder by running:
+
+```shell
+cd /etc/newrelic-infra/integrations.d
+```
+
+2. Copy the sample configuration file by running:
+
+```shell
+sudo cp nri-kafka-config.yml.sample kafka-config.yml
+```
+
+3. Edit the `kafka-config.yml` file. The following example collects metrics and inventory discovered from two JMX hosts. 
+
+```
+integrations:
+  - name: nri-kafka
+    env:
+      CLUSTER_NAME: testcluster1
+      KAFKA_VERSION: "1.0.0"
+      AUTODISCOVER_STRATEGY: zookeeper
+      ZOOKEEPER_HOSTS: '[{"host": "localhost", "port": 2181}, {"host": "localhost2", "port": 2181}]'
+      ZOOKEEPER_PATH: "/kafka-root"
+      DEFAULT_JMX_USER: username
+      DEFAULT_JMX_PASSWORD: password
+      TOPIC_MODE: all
+    interval: 15s
+    labels:
+      env: production
+      role: kafka
+    inventory_source: config/kafka
+```
+
+You can set the values for these settings in several ways:
+
+- Adding the value directly in the config file. This is the most common way.
+- Replacing the values from environment variables using the `{{ }}` notation. Read more about [using environment variable passthroughs with on-host integrations](/docs/infrastructure/install-infrastructure-agent/configuration/configure-infrastructure-agent/#passthrough) or see the example for [environment variables replacement](/docs/infrastructure/host-integrations/host-integrations-list/elasticsearch/elasticsearch-integration#envvar-replacement).
+- Using secrets management. Use this to protect sensitive information, such as passwords that would be exposed in plain text on the configuration file. For more information, see [secrets management](/docs/integrations/host-integrations/installation/secrets-management).
+
+
+The config file will likely require fine-tuning based on your specific Kafka instance. For example, you may want to use SSL or TLS to authenticate. You can find all the config options at the bottom of this doc alongside more complex config examples.
```

Comment:
Possible to link to the section? Also personal thing here, but "likely" reads weirdly to me. Super small thing, feel free to ignore.